### PR TITLE
RHBZ#1452667: Catch Anaconda Remediations parsing errors

### PR DESCRIPTION
**Pull request by jan-cerny**
The parser won't exit if it encounters something unexpected
in Ananconda remediations. Instead it will report a warning to log.

### CODE REVIEW

File: org_fedora_oscap/rule_handling.py [hunk 1]
```diff
@@ -170,10 +184,8 @@ def new_rule(self, rule):
         first_word = rule.split(None, 1)[0]
         try:
             actions[first_word](rule)
-        except KeyError:
```
Comment (yuumasato) [inline org_fedora_oscap/rule_handling.py]: We still need to catch `KeyError`. This catch is about not finding the rule in `actions` dictionary.

Comment (redhatrises) [reply org_fedora_oscap/rule_handling.py]: @jan-cerny per @yuumasato. Let's make sure to check for `KeyError`.

Comment (jan-cerny) [reply org_fedora_oscap/rule_handling.py]: thank you

Comment (jan-cerny) [reply org_fedora_oscap/rule_handling.py]: It will be here: https://github.com/OpenSCAP/oscap-anaconda-addon/pull/51

**Pull request merged**